src/material-experimental/mdc-autocomplete/_autocomplete-theme.scss

```diff
@@ -19,6 +20,12 @@
       // Note that we include this private mixin, because the public one adds
       // a bunch of styles that we aren't using for the autocomplete panel.
       @include mdc-list-base_($mat-typography-styles-query);
+
+      // MDC uses the `subtitle1` level for list items, but the spec shows `body1` as the correct
+      // level.
+      .mat-mdc-option {
+        @include mdc-typography(body1, $query: $mat-typography-styles-query);
```

Comment:
These seem to be the same between autocomplete and select. Maybe we should do it once inside the option theme instead?

Reply:
I didn't see an existing mixin for option typography

Reply:
There's `mat-mdc-option-typography`. Also now that I'm thinking about it, we likely need the same typography change for `mat-optgroup` since it uses the same size as the option.

Reply:
Oh hmm, don't know how I missed that, I'll move it

Reply:
The groups are set up to look like options, except that they aren't clickable, so the same styles can be applied.

Reply:
I think they're currently set to be subtitle1 which makes sense, after the typography levels are updated it'll wind up looking nearly identical to the options (except for the letter spacing) https://material.io/design/typography/the-type-system.html#type-scale

Reply:
Funnily enough, the link you just sent has an example of an option group which is somewhat different than what we have. Anyway, my comment was mostly to raise awareness that we might want to change the option groups as well, I don't have an opinion on what they should be changed to.
![The_type_system_-_Material_Design_-_Google_Chrome_2021-01-26_23-33-08](https://user-images.githubusercontent.com/4450522/105914669-eb74e200-602e-11eb-9107-5b18fbabba89.png)

Reply:
I created a PR based on this one that uses a config created from MDC's typography settings: https://github.com/angular/components/pull/21709 That should give an idea of how things should eventually end up looking. Though looking at it, the form-field appears to need a little more work

Reply:
It looks like they're using the `overline` level for the option group, we may want to do that too but I'll leave it for another PR.